bigquery/google/cloud/bigquery/_http.py

```diff
@@ -14,21 +14,32 @@
 
 """Create / interact with Google BigQuery connections."""
 
+import google.api_core.gapic_v1.client_info
 from google.cloud import _http
 
 from google.cloud.bigquery import __version__
 
 
-_CLIENT_INFO = _http.CLIENT_INFO_TEMPLATE.format(__version__)
-
-
 class Connection(_http.JSONConnection):
     """A connection to Google BigQuery via the JSON REST API.
 
     :type client: :class:`~google.cloud.bigquery.client.Client`
     :param client: The client that owns the current connection.
     """
 
+    def __init__(self, client, client_info=None):
```

Comment:
@tswast Need to document the `client_info` parameter in the class docstring.

Reply:
We should probably hoist this machinery into the `google.cloud._http.JSONConnection` base class.